packages/nuxt-locokit/src/runtime/services/workspace.ts

```diff
@@ -1,10 +1,11 @@
 import { SERVICES } from '@locokit/definitions'
+import { WorkspaceData, WorkspacePatch } from '@locokit/sdk'
 import { sdkClient } from './api'
 
 const ITEMS_PER_PAGE = 10
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-export async function getWorkspace(id: string, params: null | any = null) {
+export async function getWorkspace(id: string, params?: any) {
```

Comment:
Have you checked the impact of removing the default value?

Reply:
I update the signature to respect the initial one from the feathers client.

cf https://github.com/feathersjs/feathers/blob/dove/packages/feathers/src/declarations.ts#L51

By the way, I'm not sure it will be necessary to create a `services/{service}.ts` file as the sdk should be fine. 
We'll see in the future as we progress in the sdk usage.